packages/calypso-build/webpack/sass.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+const MiniCssExtractPluginWithRTL = require( 'mini-css-extract-plugin-with-rtl' );
+const WebpackRTLPlugin = require( 'webpack-rtl-plugin' );
+
+/**
+ * Return a webpack loader object containing our styling (Sass -> CSS) stack.
+ *
+ * @param {boolean}   preserveCssCustomProperties  whether Custom Properties and properties using them should be preserved in their original form
+ * @param {string[]}  includePaths                 Sass files lookup paths
+ * @param {string}    prelude                      String to prepend to each Sass file
+ *
+ * @return {Object}                                webpack loader object
+ */
+module.exports.loader = ( { preserveCssCustomProperties, includePaths, prelude } ) => ( {
+	test: /\.(sc|sa|c)ss$/,
+	use: [
+		MiniCssExtractPluginWithRTL.loader,
+		{
+			loader: require.resolve( 'css-loader' ),
+			options: {
+				importLoaders: 2,
+			},
+		},
+		{
+			loader: require.resolve( 'postcss-loader' ),
+			options: {
+				config: {
+					ctx: {
+						preserveCssCustomProperties,
+					},
+				},
+			},
+		},
+		{
+			loader: require.resolve( 'sass-loader' ),
+			options: {
+				includePaths,
+				data: prelude,
+			},
+		},
+	],
+} );
+
+/**
+ * Return an array of styling relevant webpack plugin objects.
+ *
+ * @param {String}   cssFilename  filename pattern to use for CSS files
+ * @param {Boolean}  minify       whether to minify CSS
```

Comment:
This doc is inaccurate. It describes two positional params, but the function accepts a single (unnamed) object with these properties.

I _think_ it's supposed to be like this (#)[https://stackoverflow.com/a/42898969/1432801]:

```
	 * @param {Object} _             …(gotta write something 🙄)
	 * @param {string} _.cssFilename …
	 * @param {string} _.minify Bearer …
```

https://github.com/Automattic/wp-calypso/blob/f3e8b9b75424b2bc9ede4d3fdad4eb5fd86e1363/client/jetpack-connect/signup.js#L128-L135

Reply:
Uh bummer, you're obv right. Rookie mistake on my part :roll_eyes: I'll fix.

Reply:
My go-to example in Calypso: https://github.com/Automattic/wp-calypso/blob/97569a3c821c111dba318498cdf79d45db32a217/client/lib/query-manager/index.js#L211-L226

Reply:
67b29f0